src/test/java/core/basesyntax/io/FileWriterImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.io;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FileWriterImplTest {
+
+    private FileWriterImpl fileWriter;
+    private final String filePath = "testFile.txt";
+
+    @BeforeEach
+    void setUp() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    void testWriteDataSuccessfully() throws IOException {
```

Comment:
Please, follow the method name convention
For this task use such convention: <methodUnderTest>_<state>_<expectedBehavior>

Reply:
what rules to follow ?